runtime/lua/vim/filetype.lua

```diff
@@ -635,6 +639,12 @@ local extension = {
   tssop = "tssop",
   tutor = "tutor",
   twig = "twig",
+  ts = function(name, bufnr)
+    if string.find(getline(bufnr, 1), "<%?xml") then
```

Comment:
can just be `getline(bufnr, 1):find("<%?xml")`
```suggestion
    if getline(bufnr, 1):find("<%?xml") then
```
(unless @gpanders wanted the "long form" intentionally?)

Reply:
+1 for "short form"

Reply:
> unless @gpanders wanted the "long form" intentionally?

No this is fine 👍